tests/points/test_views.py

```diff
@@ -41,6 +45,52 @@ def test_category_points_geography_not_found(self):
         self.get('category-points-geography', profile_id=self.profile.pk, profile_category_id=self.profile_category.pk, geography_code=geography.code, extra={'format': 'json'})
         self.assert_http_404_not_found()
 
+    def test_coordinates_inside_TSH_geography_for_specific_category(self):
+        call_command('loaddata', 'tests/fixtures/tsh.json', verbosity=0)
```

Comment:
Fixtures can be loaded using the fixtures field on TestCase https://docs.djangoproject.com/en/3.2/topics/testing/tools/#django.test.TransactionTestCase.fixtures